erpnext/crm/report/lead_details/test_lead_details.py

```
@@ -0,0 +1,100 @@
+		)
+
+	def test_lead_details_filter_by_status(self):
+		"""Test that filtering by status returns only matching leads."""
+		filters = frappe._dict(
+			company=self.company,
+			from_date=add_days(getdate(), -1),
+			to_date=add_days(getdate(), 1),
+			status="Open",
+		)
+
+		columns, data = execute(filters)
```

Ruff (0.15.12)

[warning] Unpacked variable `columns` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)